homeassistant/components/light/mqtt.py

```diff
@@ -266,7 +277,11 @@ def turn_on(self, **kwargs):
 
         if ATTR_COLOR_TEMP in kwargs and \
            self._topic[CONF_COLOR_TEMP_COMMAND_TOPIC] is not None:
-            color_temp = int(kwargs[ATTR_COLOR_TEMP])
+            color_temp = int(float((
+                self.templates[CONF_COLOR_TEMP+"_COMMAND"](
```

Comment:
Hey @MartinHjelmare -- doesn't CONF_COLOR_TEMP_COMMAND have to be added to https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/const.py ? Is that the change you are calling for?